src/partr.c

```diff
@@ -274,11 +274,42 @@ JL_DLLEXPORT int jl_enqueue_task(jl_task_t *task)
     return 0;
 }
 
+static int checked_running_under_rr = 0;
+static int is_running_under_rr = 0;
+#define RR_CALL_BASE 1000
+#define SYS_rrcall_check_presence (RR_CALL_BASE + 8)
```

Comment:
```suggestion
#ifdef _OS_LINUX_
static int checked_running_under_rr = 0;
static int is_running_under_rr = 0;
#define RR_CALL_BASE 1000
#define SYS_rrcall_check_presence (RR_CALL_BASE + 8)
#endif
```
(though might make sense to move these inside the `running_under_rr` function)